webapp/tests/karma/ts/services/telemetry.service.spec.ts

```diff
@@ -264,7 +264,7 @@ describe('TelemetryService', () => {
       expect(telemetryDb.query.called).to.be.false;       // NO telemetry aggregation has
       expect(metaDb.put.callCount).to.equal(0);           // been recorded yet
 
-      clock = sinon.useFakeTimers(moment(NOW).add(1, 'minutes').valueOf()); // 1 min later ...
+      clock.tick('01:00'); // 1 min later ...
```

Comment:
These changes aren't needed yet, but I was trying to get the sinon upgrade to work, and may as well keep these changes for when we finish the job...